morango/utils/syncing_utils.py

```diff
@@ -1,14 +1,93 @@
-_syncing_models = {}  # dictionary of models with morango naming identifier
```

Comment:
Maybe we should rename this file to just "syncing.py", since it's in a utils folder already.